packages/wallets/src/components/DesktopWalletsList/DesktopWalletsList.tsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
-import { useWalletAccountsList } from '@deriv/api';
+import { useAuthorize, useWalletAccountsList } from '@deriv/api';
 import { AccountsList } from '../AccountsList';
 import { WalletListCard } from '../WalletListCard';
 import { WalletsAccordion } from '../WalletsAccordion';
 import './DesktopWalletsList.scss';
 
 const DesktopWalletsList: React.FC = () => {
-    const { data: wallet_accounts_list } = useWalletAccountsList();
+    const { data } = useWalletAccountsList();
```

Comment:
Is this a convention we are following? `data` over `wallet_accounts_list`. In my previous places we have tried to avoid the word `data` in favour of more descriptive variable names?

Reply:
@markwylde-deriv We don't have a convention for these yet, Need to discuss it, For now I will revert it back to the named variable 🙇🏻